benchllm/input_types.py

```diff
@@ -1,8 +1,10 @@
 import json
-from typing import TypedDict
+from typing import TypedDict, Union
 
 from pydantic import BaseModel
 
+Json = Union[str, bool, list, dict]
```

Comment:
I don't think this is the right way of typing a JSON-serializable object/variable. I found a very interesting issue in the Python official repo: https://github.com/python/typing/issues/182.

I really like this approach:
> All major type checkers now support recursive type aliases by default, so this should largely work:

```python
JSON: TypeAlias = dict[str, "JSON"] | list["JSON"] | str | int | float | bool | None
```

What do you think?

Reply:
Interesting I will check it out, I removed int and float on purpose since it made pydantic parse things in a strange way

```python 
"2" -> 2
["2"] -> ["2"] 
```
which was kind of unexpected and hard to work with.

Reply:
https://github.com/pydantic/pydantic/discussions/5779

Seems like Pydantic does not yet support recursive TypeAlias